engine/api/src/main/scala/pl/touk/nussknacker/engine/api/process/WithCategories.scala

```diff
@@ -30,10 +30,10 @@ object WithCategories {
 }
 
 object ParameterConfig {
-  val empty: ParameterConfig = ParameterConfig(None, None)
+  val empty: ParameterConfig = ParameterConfig(None, None, None)
 }
 
-@JsonCodec case class ParameterConfig(defaultValue: Option[String], editor: Option[ParameterEditor])
+@JsonCodec case class ParameterConfig(defaultValue: Option[String], editor: Option[ParameterEditor], validators: Option[List[ParameterValidator]])
```

Comment:
Why it is `Option[List[]]` instead of just `List[]`?

Reply:
It can be undefined in config file. Then it maps to None.

You could also define [] list in config file. Then in maps to empty list

Reply:
Ok, now I see that it is used in config file.